src/main/java/org/hyperledger/aries/webhook/WebSocketListener.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2020-2022 - for information on the respective copyright owner
+ * see the NOTICE file and/or the repository at
+ * https://github.com/hyperledger-labs/acapy-java-client
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.aries.webhook;
+
+import org.hyperledger.aries.config.GsonConfig;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonSyntaxException;
+
+import lombok.extern.slf4j.Slf4j;
+import okhttp3.Response;
+import okhttp3.WebSocket;
+
+@Slf4j
+public class WebSocketListener extends okhttp3.WebSocketListener {
```

Comment:
Wouldn't it be cleaner to call this class (AcaPy|Aries|Base)WebsocketListener and to make it abstract as well so that clients are free to overwrite the onOpen() method for example without loosing the onMessage() implementation? If so the WebSocketEventHandler above would make sense as a DefaultWebSocketListener extending this class.